feat: add `array/base/cuevery-by-right` (#2802)
stdlib-js/stdlib: lib/node_modules/@stdlib/array/base/cuevery-by-right/docs/repl.txt

{{alias}}( x, predicate[, thisArg] )

>     Cumulatively test whether every array element in a provided array

Reviewer: ```suggestion
    Cumulatively tests whether every array element in a provided array
```

>     pass a test implemented by a predicate function, while iterating from

Reviewer: ```suggestion
    passes a test implemented by a predicate function, while iterating from
```

    right-to-left.

    The predicate function is provided three arguments:

    - `value`: current array element.
    - `index`: current array element index.
    - `arr`: the input array.

    Parameters
    ----------
    x: ArrayLikeObject
        Input array.

    predicate: Function
        Predicate function.

    thisArg: any (optional)
        Execution context.

    Returns
    -------
    out: Array
        Output array.

    Examples
    --------
    > function fcn( v ) { return ( v > 0 ); };
    > var x = [ 0, 0, 1, 1, 1 ];
    > var y = {{alias}}( x, fcn )
    [ true, true, true, false, false ]

{{alias}}.assign( x, out, stride, offset, predicate[, thisArg] )

>     Cumulatively test whether every array element in a provided array
>     pass a test implemented by a predicate function, while iterating from
>     right-to-left and assign the results to the provided output array.

Reviewer: ```suggestion
    Cumulatively tests whether every array element in a provided array
    passes a test implemented by a predicate function, while iterating from
    right-to-left, and assigns the results to the provided output array.
```
